packages/fxa-settings/src/models/contexts/NimbusContext.ts

```diff
@@ -73,21 +73,29 @@ export function NimbusProvider({ children }: NimbusProviderProps) {
   const [loading, setLoading] = useState<boolean>(false);
   const [error, setError] = useState<Error | undefined>(undefined);
 
-  useEffect(() => {
+  // Monitor currentAccountUid changes in localStorage (reacts to same-tab and cross-tab changes)
+  const currentAccountUid = useLocalStorageSync('currentAccountUid');
+  const accounts = useLocalStorageSync('accounts');
+
+  // Get metricsEnabled from accounts object, safely handling undefined values
+  const accountMetricsEnabled = useMemo(() => {
+    if (!currentAccountUid || !accounts) {
+      return undefined;
```

Comment:
If I'm not signed into an account, this value is undefined and so the Nimbus experiments don't get fetched. Should this be `true` instead?

Also, do you plan to address the 1. and 2. points of the ticket in this PR? I think this only covers the 3rd case, but the other 2 should be easy at least if you want to do them here or in a quick follow up.